guide/C/ch_basics.xml

```diff
@@ -619,11 +619,151 @@
+  				<entry align="center" namest="c4mysql"      > N/A - 2 </entry>
+  				<entry align="center" namest="c5postgresql" > N/A - 2 </entry>
+                </row>
+
+                <row>
+                  <entry >Additional software</entry>
+  				<entry align="center" namest="c2xml"        > None </entry>
+  				<entry align="center" namest="c3sqlite"     > None </entry>
+  				<entry align="center" namest="c4mysql"      > MySQL </entry>
+  				<entry align="center" namest="c5postgresql" > PostgreSQL </entry>
+                </row>
+
+                <row>
+                  <entry >Additional expertise</entry>
+  				<entry align="center" namest="c2xml"        > None </entry>
+  				<entry align="center" namest="c3sqlite"     > None </entry>
+  				<entry align="center" namest="c4mysql"      > DBMS </entry>
+  				<entry align="center" namest="c5postgresql" > DBMS </entry>
+                </row>
+
+                <row>
+                  <entry >Compression</entry>
+  				<entry align="center" namest="c2xml"        > Y </entry>
+  				<entry align="center" namest="c3sqlite"     > N </entry>
+  				<entry align="center" namest="c4mysql"      > N </entry>
+  				<entry align="center" namest="c5postgresql" > N </entry>
+               </row>
+
+                <row>
+                  <entry >Save on command</entry>
+  				<entry align="center" namest="c2xml"        > Y </entry>
+  				<entry align="center" namest="c3sqlite"     > N </entry>
+  				<entry align="center" namest="c4mysql"      > N </entry>
+  				<entry align="center" namest="c5postgresql" > N </entry>
+                </row>
+
+                <row>
+                  <entry >Save on commit</entry>
+  				<entry align="center" namest="c2xml"        > N </entry>
+  				<entry align="center" namest="c3sqlite"     > Y </entry>
+  				<entry align="center" namest="c4mysql"      > Y </entry>
+  				<entry align="center" namest="c5postgresql" > Y </entry>
+                </row>
+
+                <row>
+                  <entry >Uses log files</entry>
+  				<entry align="center" namest="c2xml"        > Y </entry>
+  				<entry align="center" namest="c3sqlite"     > N </entry>
+  				<entry align="center" namest="c4mysql"      > N </entry>
+  				<entry align="center" namest="c5postgresql" > N </entry>
+                </row>
+
+                <row>
+                  <entry >Multi-user</entry>
+  				<entry align="center" namest="c2xml"        > N </entry>
+  				<entry align="center" namest="c3sqlite"     > N </entry>
+  				<entry align="center" namest="c4mysql"      > N </entry>
+  				<entry align="center" namest="c5postgresql" > N </entry>
+                </row>
+                </tbody>
+              </tgroup>
+            </table>
+			<para>1 - SQLite relies on libdbi. However, it is available by 
+			default on Mac OS and Windows. Linux users may need to manually 
+			install libdbi drivers for SQLite.</para>
```

Comment:
I would formulate it the other way:
libdbi-dbd-sqlite3 is required (in the table)
And In the footnote it is part of our Macos and Windows bundles/packages.

Reasons: The sqlite dbd is not part of Windows and Macos, but of **our** bundles. The user could accidently remove the driver. There are other bundles in the universe i.e. MacPorts where we do not control the packaging.

Reply:
I'll look into this some more. I suspect I'll go with your suggestion; I just have to give it a better look.

Reply:
Again, this is the overview chapter. Let's keep it simple and put the technical details in the wiki.

Reply:
This overlaps a bit with my comments above. I agree we should keep this light and have made suggestions to keep a reasonable balance.